### PR TITLE
passwordMode: add keyboard layout switcher for password entries

### PR DESCRIPTION
**Pull request by GeorgesStavracas**
Fallout from the last rebase.

https://phabricator.endlessm.com/T29997

### CODE REVIEW

**Comment by GeorgesStavracas** [comment]
I couldn't figure out yet why sometimes the current layout name shows as a `.` in dialogs, but I guess that would be unrelated to this patch?

**Comment by GeorgesStavracas** [comment]
Fixed

File: js/ui/status/keyboard.js [hunk 1]
```diff
@@ -488,6 +497,45 @@ var InputSourceManager = class {
             this._updateMruSettings();
     }
 
+    get PasswordModeEnabled() {
+        return this.passwordModeEnabled;
```
Comment (andrunko) [inline js/ui/status/keyboard.js:501]: This look strange to me, why do we have 2 getters and setters for this property? And why one of them uses `this._passwordModeEnabled` while the other uses `this.passwordModeEnabled`? Same is valid for the setter below.

Comment (GeorgesStavracas) [reply js/ui/status/keyboard.js:501]: One is the D-Bus property implementation, the other is the JavaScript property.

Comment (andrunko) [reply js/ui/status/keyboard.js:501]: I see, thanks.

**Pull request merged**